mlir-sycl/test/Conversion/SYCLToLLVM/sycl-constructor-to-llvm.mlir

```diff
@@ -4,14 +4,14 @@
 // Constructors for sycl::accessor<t, d, m, t, p>::accessor()
 //===-------------------------------------------------------------------------------------------------===//
 
-!sycl_accessor_1_i32_read_write_global_buffer = !sycl.accessor<[1, i32, read_write, global_buffer], (!sycl.accessor_impl_device<[1], (!sycl.id<1>, !sycl.range<1>, !sycl.range<1>)>, !llvm.struct<(ptr<i32, 1>)>)>
+!sycl_accessor_1_ = !sycl.accessor<[1, i32, read_write, global_buffer], (!sycl.accessor_impl_device<[1], (!sycl.id<1>, !sycl.range<1>, !sycl.range<1>)>, !llvm.struct<(ptr<i32, 1>)>)>
```

Comment:
I actually think that the old naming convention is better because the old name did report exactly the type of accessor we deal with (includes the access type and the element type).  Can we go back to that naming convention pls? 

Personally, the fact the type alias has a long name does not bother me much because it contains useful info. If we want to shorten the name then how about shortening it like:   `sycl_acc_1_i32_rw_gb_1_`.

Reply:
Using ` sycl_accessor_1_i32_rw_gb` now.